resources/mediasearch-vue/components/SearchResults.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-	<div v-bind:class="'wbmi-mediasearch-results--' + mediaType"
-		class="wbmi-mediasearch-results">
+	<div v-bind:class="'wbmi-media-search-results--' + mediaType"
```

Comment:
When we do include media search in class names and other stuff, i think we should hyphenate/treat it as two separate words for clarity

Reply:
Do you prefer to avoid having both `v-bind:class` and `class` on the same element? I don't mind separating out the dynamic and the static classes this way (especially if there are lots of static ones – we generally just care about what can change).